embedding/csharp/Tizen.Flutter.Embedding/Plugins/FlutterDotnetPluginBinding.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+using static Tizen.Flutter.Embedding.Interop;
+
+namespace Tizen.Flutter.Embedding
+{
+    internal class FlutterDotnetPluginBinding : IFlutterPluginBinding
```

Comment:
Any suggestions do you have?

I just wanted to show that those internal classes are for dotnet plugins. I think `Flutter` prefix can be omitted. 
But, in case of `IFlutterPlugin`, I think `Flutter` prefix is necessary because `IPlugin` is too abstract. .. :(

Reply:
You can remove either or none. "FlutterDotnet" looked just too verbose for me but I'm okay with it.